module/input/FakeCamera/src/FakeCamera.cpp

```diff
@@ -82,7 +82,7 @@ namespace module::input {
 
                     // Traverse specified directory and locate all image/lens pairs
                     for (const auto& p : std::filesystem::recursive_directory_iterator(config.image_folder)) {
-                        if (utility::strutil::endsWith(p.path(), std::vector<std::string>{".jpg", ".jpeg"})) {
+                        if (utility::strutil::ends_with(p.path(), std::vector<std::string>{".jpg", ".jpeg"})) {
```

Comment:
Seems unrelated to this PR

Reply:
StreamReactor uses `dedent` which relies on other functions in strutil like `join` and the like. I will try to be consistent but I think I will have to definitely add stuff into strutil and make it a part of this pr otherwise it wont compile